login.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { Text, View, StyleSheet, SectionList, Image, Button, TouchableOpacity, ActivityIndicator } from 'react-native';
+import { TextInput } from 'react-native-gesture-handler';
+import { color } from 'react-native-reanimated';
+import { AsyncStorage } from 'react-native';
+
+const axios = require('axios')
+
+class LoginScreen extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            detail: {
+                username: '',
+                password: '',
+            },
+            loading: false,
+            error: false,
+        }
+    }
+    handleLogin = () => {
+        this.setState({ loading: true })
+        axios({
+            method: 'post',
+            url: 'http://10.0.2.2:5484/auth/login',
```

Comment:
Create utility/helper function export endpoint constant. don't create hardcode string in the component, create constant file and use it